xlators/features/shard/src/shard.c

```diff
@@ -16,6 +16,8 @@
 #include <glusterfs/defaults.h>
 #include <glusterfs/statedump.h>
 
+#define SHARD_PATH_MAX sizeof(GF_SHARD_DIR) + GF_UUID_BUF_SIZE + 16
```

Comment:
To avoid problems, always put macro computations between parenthesis.

Reply:
sure